econml/_cate_estimator.py

```diff
@@ -904,8 +962,8 @@ def summary(self, alpha=0.1, value=0, decimals=3, feature_names=None, treatment_
             converted to various output formats.
         """
         # Get input names
-        treatment_names = self._input_names["treatment_names"] if treatment_names is None else treatment_names
-        output_names = self._input_names["output_names"] if output_names is None else output_names
+        treatment_names = self.cate_treatment_names() if treatment_names is None else treatment_names
```

Comment:
Once the `cate_treatment_names` take an optional input, with default None, this line should just be: `self.cate_treatment_names(treatment_names)` and `self.cate_output_names(output_names)`

Reply:
Fixed!